src/bci_build/package/kubectl.py

```diff
@@ -0,0 +1,60 @@
+"""Kubectl BCI container"""
+
+from bci_build.container_attributes import SupportLevel
+from bci_build.os_version import CAN_BE_LATEST_OS_VERSION
+from bci_build.os_version import OsVersion
+from bci_build.package import ApplicationStackContainer
+from bci_build.package import ParseVersion
+from bci_build.package import Replacement
+from bci_build.package.helpers import generate_from_image_tag
+from bci_build.package.versions import format_version
+
+KUBECTL_CONTAINERS = []
+
+KUBECTL_CONTAINERS = [
+    ApplicationStackContainer(
+        name="kubectl",
```

Comment:
We must set a `package_name`. ATM all containers are called `kubectl-image` and we only create one image with the latest one created.
```suggestion
        name="kubectl",
        package_name=f"kubectl-{ver}-image",
```

Reply:
fixed.